charts-source/capi/Chart.yaml

```
@@ -4,19 +4,19 @@ description: capi-controller-manager compatible with Rancher Provisioning
 home: https://github.com/rancher/provisioning/blob/main/charts/capi/
 sources:
   - "https://github.com/rancher/provisioning/blob/main/charts/capi/"
-version: 0.3.0
-appVersion: "1.7.3"
+version: 0.4.0
+appVersion: "1.8.3"
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
   catalog.cattle.io/display-name: Rancher Provisioning CAPI Controller Manager
-  catalog.cattle.io/kube-version: '>= 1.27.0-0 < 1.31.0-0'
+  catalog.cattle.io/kube-version: '>= 1.27.0-0 < 1.32.0-0'
   catalog.cattle.io/namespace: cattle-provisioning-capi-system
   catalog.cattle.io/release-name: rancher-provisioning-capi
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: apps.deployment/v1
-  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.11.0-0'
```

Any reason we want to keep 2.9.x in the range? (Note this condition doesn't really matter much as it's only used by UI for user-deployed charts but it still makes sense to keep things clean here).

```suggestion
  catalog.cattle.io/rancher-version: '>= 2.10.0-0 < 2.11.0-0'
```